app/scripts/controllers/mmi-controller.ts

```diff
@@ -790,6 +790,13 @@ export default class MMIController extends EventEmitter {
     const isCustodial = Boolean(accountDetails);
     const updatedMsgParams = { ...msgParams, deferSetAsSigned: isCustodial };
 
+    if (
+      !req.method.includes('eth_signTypedData') &&
+      !req.method.includes('personal_sign')
```

Comment:
personal_sign can probably be a `===` check, but tbh i don't love the `req.method.includes('eth_signTypedData')` here in general haha